lnrpc/routerrpc/router_backend.go

```diff
@@ -231,8 +231,8 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 	// If we have any TLV records destined for the final hop, then we'll
 	// attempt to decode them now into a form that the router can more
 	// easily manipulate.
-	err = ValidateCustomRecords(in.DestCustomRecords)
-	if err != nil {
+	customRecords := record.CustomSet(in.DestCustomRecords)
```

Comment:
I guess it is not a problem that copy is by reference here?

Reply:
No, it is no problem. Those rpc parameters are not used anywhere else.